mapswipe_workers/mapswipe_workers/definitions.py

```diff
@@ -16,6 +16,8 @@
 OSM_API_LINK = "https://www.openstreetmap.org/api/0.6/"
 OSMCHA_API_LINK = "https://osmcha.org/api/v1/"
 OSMCHA_API_KEY = os.environ["OSMCHA_API_KEY"]
+MAPILLARY_API_LINK = "https://tiles.mapillary.com/maps/vtp/mly1_computed_public/2/"
+MAPILLARY_API_KEY = os.environ["MAPILLARY_API_KEY"]
```

Comment:
I'm confused above in the example.env you specify `MAPILLARY_ACCESS_KEY` , here you use `MAPILLARY_API_KEY`